src/fr/introduction/accessible-image.html

```diff
@@ -0,0 +1,32 @@
+---
+title: Concevoir des images accessibles
+layout: layouts/base.njk
+description: Grâce à ce diagramme, vous apprendrez à faire le meilleur choix pour concevoir des images accessibles.
+---
+        <div class="row">
+            <div class="col-md-10 col-xs-offset-1"><img src="../../../images/fr/image-accessible.jpg"
```

Comment:
image path is wrong.
https://deploy-preview-144--a11ycanada.netlify.app/fr/introduction/accessible-image/